appcues/src/main/java/com/appcues/ui/primitive/BoxPrimitive.kt

```diff
@@ -17,33 +14,17 @@ import com.appcues.data.model.styling.ComponentColor
 import com.appcues.data.model.styling.ComponentStyle
 import com.appcues.data.model.styling.ComponentStyle.ComponentHorizontalAlignment
 import com.appcues.data.model.styling.ComponentStyle.ComponentVerticalAlignment
-import com.appcues.ui.LocalAppcuesActionDelegate
-import com.appcues.ui.LocalAppcuesActions
-import com.appcues.ui.extensions.Compose
-import com.appcues.ui.extensions.PrimitiveGestureProperties
 import com.appcues.ui.extensions.getBoxAlignment
 import com.appcues.ui.extensions.getTextStyle
-import com.appcues.ui.extensions.primitiveStyle
-import com.appcues.ui.theme.AppcuesPreview
+import com.appcues.ui.theme.AppcuesPreviewPrimitive
 import java.util.UUID
 
 @Composable
-internal fun BoxPrimitive.Compose() {
+internal fun BoxPrimitive.Compose(modifier: Modifier) {
```

Comment:
all primitives now receives a modifier it should apply to the parent Compose (in this case is the box on line 24), this ensures that Modifier.innerStylePrimitive is being applied without code duplication.